node/cmd/guardiand/node.go

```diff
@@ -352,6 +359,8 @@ func init() {
 	telemetryProject = NodeCmd.Flags().String("telemetryProject", defaultTelemetryProject,
 		"Google Cloud Project to use for Telemetry logging")
```

Comment:
I guess this isn't just for the Google Cloud Project anymore.

Reply:
No longer using this parameter in Loki, so this string is now/still correct.